WEBHOOKS.md

```diff
@@ -119,6 +119,90 @@ The following event types have been observed:
 - `CombatBigLootDaily`
 - `SkillingParty`
 
+### Clan Action Webhooks
+
+Clan action webhooks are triggered by various clan-related activities and provide real-time notifications about clan activities.
+
+- **Method**: POST
+- **Path**: `/clan`
+- **Body Parameters**: Varies by action type (see individual actions below)
+
+#### Available Clan Actions
+
+The following clan action webhooks are implemented:
+
+##### Member Management
+- **`member-logged-in`** - Triggered when a clan member logs into the game
+  - Parameters: `username`
+- **`member-logged-out`** - Triggered when a clan member logs out of the game
+  - Parameters: `username`
+- **`member-joined-clan`** - Triggered when a player joins the clan
+  - Parameters: `username`
+- **`member-left-clan`** - Triggered when a player leaves the clan
+  - Parameters: `username`
+- **`member-promoted`** - Triggered when a clan member is promoted
+  - Parameters: `member`, `new_rank`, `username`
+- **`member-demoted`** - Triggered when a clan member is demoted
+  - Parameters: `member`, `new_rank`, `username`
+- **`member-kicked`** - Triggered when a clan member is kicked from the clan
+  - Parameters: `member`, `username`
+
+##### Clan Applications
+- **`application-received`** - Triggered when a player applies to join the clan
+  - Parameters: `player_applying`, `player_total_level`, `message`
+
+##### Clan Resources & Items
+- **`skilling-ticket-received`** - Triggered when a clan member receives a skilling ticket
+  - Parameters: `skill`, `amount`, `username`
+- **`item-sent-to-vault`** - Triggered when a clan member sends an item to the clan vault
+  - Parameters: `item_id`, `item_name`, `amount`, `username`
+- **`items-sent-to-vault`** - Triggered when a clan member sends multiple items to the clan vault
+  - Parameters: `items` (serialized JSON array), `username`
+- **`items-withdrawn-from-vault`** - Triggered when a clan member withdraws an item from the clan vault
```

Comment:
Shouldn't this be `item-withdrawn-from-vault`, as specified in `ClanActionWebhooks`.